src/global_distrib/mod_global_distrib.erl

```diff
@@ -76,17 +76,20 @@ maybe_reroute({From, To, Acc0, Packet} = FPacket) ->
     Acc = maybe_initialize_metadata(Acc0),
     LocalHost = opt(local_host),
     GlobalHost = opt(global_host),
-    case lookup_recipients_host(To, LocalHost, GlobalHost) of
+    case lookup_recipients_host(get_metadata(Acc, target_host_override, undefined),
```

Comment:
Can we have it on a separate line?

```erlang
TargetHost = get_metadata(Acc, target_host_override, undefined),
```

It still will be two lines long :D

Reply:
target_host_override should be documented, I have no idea what is for just by looking at the code.

Reply:
and why it can be undefined should be described too.